public/css/app.css

```diff
@@ -0,0 +1,72 @@
+@import url('https://fonts.googleapis.com/css?family=Fira+Sans:400,600,700');
```

Comment:
Could we potentially copy and host this resource ourselves?

Reply:
Not sure if it'd give us all the weights we want, but it looks like Mozilla does have some sort of Fira CDN:

Per https://github.com/mozilla/Fira#usage,

> **Usage**
> Use this font on your website!
>
> ```html
> <link rel="stylesheet" href="https://code.cdn.mozilla.net/fonts/fira.css">
> ```